config/locales/diaspora/en.yml

```diff
@@ -1308,6 +1308,40 @@ en:
       email_confirmed: "Email %{email} activated"
       email_not_confirmed: "Email could not be activated. Wrong link?"
 
+  two_factor_auth:
+    title: "Two factor authentication"
+    activated:
```

Comment:
I would use 'two-factor authentication' here and throughout. The hyphen is good style when you have compound adjectives, as in this phrase.